internal/runtime/runtime.go

```diff
@@ -29,26 +30,29 @@ import (
 	"github.com/ugorji/go/codec"
 )
 
+const unmarshalErrorMessage = "Unable to %s unmarshal message payload"
```

Comment:
I think this could read a bit better: `"Unable to unmarshal message payload as %s"`